meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/cryptfs_tpm2

```diff
@@ -7,7 +7,7 @@ cryptfs_check_tpm2() {
 	# Check for SecureBoot support as PCR 7 differs based on its state
 	efi_secure=`efivar --name=8be4df61-93ca-11d2-aa0d-00e098032b8c-SecureBoot --print-decimal`
 	efi_mode=`efivar --name=8be4df61-93ca-11d2-aa0d-00e098032b8c-SetupMode --print-decimal`
-	if [ "${efi_secure}" != "1" ] || [ "${efi_mode}" != "0" ]; then
+	if [ "${efi_secure}" -ne 1 ] || [ "${efi_mode}" -ne 0 ]; then
```

Comment:
I think you should drop quotation marks for the first operand, same in `meta-lmp-base/recipes-core/initrdscripts/initramfs-module-install-efi/init-install-efi.sh`

Reply:
it doesnt really matter in this particular case - all bash variables are strings anyway. it is safer and good practice to keep the quotes in case there are spaces and so on...